__tests__/app.test.js

```diff
@@ -55,3 +55,32 @@ describe('GET /api', () => {
 
     })
 })
+
+describe('GET /api/articles/:article_id', () => {
+    test('status:200, responds with a JSON object containing specified article which has the following properties: author, title, article_id, body, topic, created_at, votes, article_img_url', () => {
+        return request(app)
+        .get('/api/articles/1')
+        .expect(200)
+        .then(({body}) => {
+
+        expect(body.article).toMatchObject({
+            author: expect.any(String),
+            title: expect.any(String),
+            article_id: expect.any(Number),
+            body: expect.any(String),
+            topic: expect.any(String),
+            created_at: expect.any(String),
+            votes: expect.any(Number),
+            article_img_url: expect.any(String)
+            })
```

Comment:
Great first test, however because the endpoint is specifically looking for an article where the article_id is 1 (/api/articles/1) we should also assert that this is what we get back from the response body as well, instead of just checking that it is any number.